config.json.sample

```diff
@@ -0,0 +1,3 @@
+{
+    "featureManagerIsFullZoneProvisioningEnabled": false
+}
```

Comment:
I think the only ones they should care about are `debug`, `locale`, and `featureManagerIsFullZoneProvisioningEnabled` though.  We don't _want_ them overwriting the cards but we don't necessarily care if they do.